frontend/src/components/Calendar/index.tsx

```diff
@@ -178,62 +109,119 @@ export const Calendar: React.FC<Props> = ({
-          <IconButton onClick={() => onChangeMonth(1)} size="large">
-            <NavigateNext />
-          </IconButton>
-        </Hidden>
-      </Stack>
-      <Stack spacing={5} direction="row" alignItems="flex-start">
-        <CalendarTable month={selectedMonth} onChangeMonth={onChangeMonth}>
+      <Typography variant="h5" align="center">
+        {title}
+      </Typography>
+      <Stack
+        spacing={5}
+        direction="row"
+        alignItems="flex-start"
+        /**
+         * We add some custom styles for the calendar days to make it look more like a booking calendar.
+         * The class names below are utilized by `CalendarDay`.
+         */
+        sx={{
+          "& .booking-fromDate": {
+            borderRadius: (theme) => `${theme.shape.borderRadius * 2}px 0px 0px ${theme.shape.borderRadius * 2}px`,
+            bgcolor: (theme) => theme.vars.palette.primary.light,
+          },
+          "& .booking-toDate": {
+            borderRadius: (theme) => `0px ${theme.shape.borderRadius * 2}px ${theme.shape.borderRadius * 2}px 0px`,
+            bgcolor: (theme) => theme.vars.palette.primary.light,
+          },
+          "& .booking-inSelectedRange": {
+            bgcolor: (theme) => theme.vars.palette.primary.dark,
+            borderRadius: 0,
+          },
+          "& .booking-inSelectedRange.Mui-disabled": {
+            bgcolor: (theme) => theme.vars.palette.action.disabledBackground,
+          },
+        }}
+      >
+        <CalendarTable
+          month={selectedMonth}
+          previousMonthIcon={
+            <IconButton onClick={() => onChangeMonth(-1)}>
+              <NavigateBefore />
+            </IconButton>
+          }
+          nextMonthIcon={
+            <Box visibility={{ xs: "visible", lg: "hidden" }}>
+              <IconButton onClick={() => onChangeMonth(1)}>
+                <NavigateNext />
+              </IconButton>
+            </Box>
+          }
+        >
           {getRows(selectedMonth).map((row, index) => (
-            <CalendarRow key={index} index={index}>
+            <CalendarRow key={index}>
               {row.map((date) => (
-                <CalendarDay
-                  value={date.date()}
-                  isFromDate={selectedFromDay ? date.isSame(selectedFromDay, "day") : false}
-                  isToDate={selectedToDay ? date.isSame(selectedToDay, "day") : false}
-                  onClick={() => handleDateClicked(date, selectedMonth)}
-                  isDisabled={isDisabled(date)}
-                  isHidden={isHidden(date, selectedMonth)}
-                  isInRange={range.includes(date.format(DATE_FORMAT))}
-                  isInvalidRange={!isRangeValid}
-                  key={date.format(DATE_FORMAT)}
-                />
+                <>
+                  {date && (
+                    <CalendarDay
+                      value={date.date()}
+                      isFromDate={startDate && date.isSame(startDate, "day")}
```

Comment:
I found this useful in something else and was wondering, why do we need the "startDate &&" as opposed to just writing date.isSame(startDate, "day")